components/fxa-client/android/build.gradle

```diff
@@ -109,7 +109,7 @@ afterEvaluate {
             productFlavor += "${it.name.capitalize()}"
         }
         def buildType = "${variant.buildType.name.capitalize()}"
-        tasks["generate${productFlavor}${buildType}Assets"].dependsOn(project(':full-megazord').tasks["cargoBuild"])
+        tasks["merge${productFlavor}${buildType}JniLibFolders"].dependsOn(project(':full-megazord').tasks["cargoBuild"])
```

Comment:
Seeing this, I wonder if we still need this in the individual projects, since they don't bundle any rust code anymore (and instead always depend on the relevant megazord). But that can be for another time.

Reply:
> Seeing this, I wonder if we still need this in the individual projects, since they don't bundle any rust code anymore (and instead always depend on the relevant megazord). But that can be for another time.

Then almost certainly not, we should be able to drop the stanzas.  Try it and see?